config/webpack.dev-stage.config.js

```diff
@@ -124,28 +125,25 @@ module.exports = merge(commonConfig, {
       },
       {
         test: /\.(jpe?g|png|gif)(\?v=\d+\.\d+\.\d+)?$/,
-        use: [
-          'file-loader',
-          {
-            loader: 'image-webpack-loader',
-            options: {
-              mozjpeg: {
-                progressive: true,
-                quality: 65,
-              },
-              gifsicle: {
-                interlaced: false,
-              },
-              pngquant: {
-                quality: [0.65, 0.90],
-                speed: 4,
-              },
-            },
-          },
-        ],
+        loader: 'file-loader',
       },
     ],
   },
+  optimization: {
+    minimizer: [
+      '...',
+      new ImageMinimizerPlugin({
+        implementation: ImageMinimizerPlugin.sharpMinify,
+        options: {
+          encodeOptions: {
+            ...['png', 'jpeg', 'jpg'].reduce((accumulator, value) => (
```

Comment:
I do see gif as part of encode options
```
type SharpEncodeOptions = {
  avif?: import("sharp").AvifOptions | undefined;
  gif?: import("sharp").GifOptions | undefined;
  heif?: import("sharp").HeifOptions | undefined;
  jpeg?: import("sharp").JpegOptions | undefined;
  jpg?: import("sharp").JpegOptions | undefined;
  png?: import("sharp").PngOptions | undefined;
  webp?: import("sharp").WebpOptions | undefined;
};
```

**GifOptions (with defaults)**
```
/** Number of animation iterations, a value between 0 and 65535. Use 0 for infinite animation. (optional, default 0) */
loop?: number | undefined;
/** delay(s) between animation frames (in milliseconds), each value between 0 and 65535. (optional) */
delay?: number | number[] | undefined;
/** Maximum number of palette entries, including transparency, between 2 and 256 (optional, default 256) */
colours?: number | undefined;
/** Alternative Spelling of "colours". Maximum number of palette entries, including transparency, between 2 and 256 (optional, default 256) */
colors?: number | undefined;
/** Level of CPU effort to reduce file size, between 1 (fastest) and 10 (slowest) (optional, default 7) */
effort?: number | undefined;
/** Level of Floyd-Steinberg error diffusion, between 0 (least) and 1 (most) (optional, default 1.0) */
dither?: number | undefined;
```

Reply:
@BilalQamar95 Am I reading correctly that we'd need to explicitly configure it to minify gif images?  We should probably do that since the existing implementation does so.